util/container.go

```diff
@@ -155,6 +166,10 @@ func ContainerRun(ctx context.Context, cli *client.Client, clusterName string, k
 				addlMysqldFlags := []string{
 					fmt.Sprintf("--server-id=%d", n),
 				}
+				// MariaDB needs some special care here for GTID configuration
+				if globals.MysqlFlavor == "mariadb" {
+					addlMysqldFlags = append(addlMysqldFlags, fmt.Sprintf("--gtid_domain_id=%d", n), fmt.Sprintf("--log-bin=%s", n),)
```

Comment:
What are we trying to do here? The linter doesn't like the formats. Are we trying to set both of these config options to YES, TRUE, or something else?

Reply:
I was trying to give it a value, --gtid_domain_id needs to be an integer, so I noticed you have %d, n in the code to give it a number (this will be either '1' or '2'. 

The --log-bin needs a string, so I modified the code again and this time gave it the hostname, so it's a string.

(Still learning go)

Reply:
I'll install a go linter on my editor so I can check this before submitting code.

Reply:
OK, but AFAIUI the gtid_domain is a failure domain and it should be the same for any instances in the same replication group (and beyond that ~ datacenter). We should probably set it to 1 for all of instances. I don't think we want it to be different here.

Reply:
Okay I read up more on this, so it looks like you're right you would want gtid_ domain to be the same for your replication group. Typically this setting is used for advanced MMM configurations; so I think we can get rid of this and just use 0 (default). 

Alternatively, we could also look to see if a Vitess cluster is already running and increment that value by 1; but that might be a bit overkill. Considering vtmanager is more for quick spin-up and test environments.